apps/web/pages/auth/setup.tsx

```diff
@@ -185,7 +189,7 @@ const SetupFormStep1 = (props: { setIsLoading: (val: boolean) => void }) => {
                   formMethods.setValue("password", e.target.value);
                   await formMethods.trigger("password");
                 }}
-                hintErrors={["caplow", "min", "num"]}
+                hintErrors={["caplow", "admin_min", "num"]}
```

Comment:
Should "min" also still remain as a hint?

Reply:
min property only checks for 8 characters or more and also displays 8 characters "password_hint_min": "Minimum 8 characters long".  check HintsOrErrors.tsx file once.